package.json

```diff
@@ -0,0 +1,68 @@
+{
+  "name": "jest-elasticsearch",
+  "version": "1.0.0",
+  "description": "Run your tests using jest & elasticsearch-local",
+  "keywords": [
+    "jest",
+    "jest environment",
+    "jest preset",
+    "elasticsearch",
+    "elasticsearch local",
+    "elasticsearch-local"
+  ],
+  "repository": "shelfio/jest-elasticsearch",
+  "license": "MIT",
+  "author": {
+    "name": "Andrii Petlovanyi",
+    "email": "andrii.petlovanyi@shelf.io",
+    "url": "shelf.io"
+  },
+  "files": [
+    "lib"
+  ],
+  "main": "lib",
+  "scripts": {
+    "coverage": "jest --coverage",
+    "lint": "eslint . --ext .js,.json --fix",
+    "lint:ci": "eslint . --ext .js,.json --quiet",
+    "test": "jest src"
+  },
+  "husky": {
+    "hooks": {
+      "pre-commit": "lint-staged"
+    }
+  },
+  "lint-staged": {
+    "*.js": [
+      "eslint --fix",
+      "git add"
+    ],
+    "*.{html,json,md,yml}": [
+      "prettier --write",
+      "git add"
+    ]
+  },
+  "jest": {
+    "preset": "./jest-preset.js"
+  },
+  "dependencies": {
+    "@elastic/elasticsearch": "7.5.0",
+    "@shelf/quotation-marks-regex": "0.1.0"
+  },
+  "devDependencies": {
+    "@shelf/elasticsearch-local": "0.2.0",
```

Comment:
Looks like this is not a `dev` dependency but a prod dependency

W/o this lib jest preset won't work